crates/ruff_python_parser/src/lexer.rs

```diff
@@ -35,6 +36,8 @@ mod indentation;
 
 #[deprecated]
 pub fn lex(_source: &str, _mode: Mode) {}
+#[deprecated]
+pub fn lex_starts_at(_source: &str, _mode: Mode, _offset: TextSize) {}
```

Comment:
These are here just to help me with finding the remaining references and will be removed at the end.